resources/public/api/conf/5.0/foreign_property_accounting_adjustments_submit.yaml

```diff
@@ -77,15 +77,6 @@ post:
           $ref: ./common/headers.yaml#/components/parameters/sunsetHeader
         Link:
           $ref: ./common/headers.yaml#/components/parameters/linkHeader
-      content:
```

Comment:
need to remove hateoas links for the other endpoints OAS as well, in schemas and examples